### PR TITLE
fix(#2239): Updated package versions to use NPM API

### PR DESCRIPTION
**Pull request by ArakTaiRoth**
In the screen Get Started --> Support --> Report a Bug the versions listed should now be pulled down via an API call instead of statically entered.

### CODE REVIEW

File: src/routes/get-started/ReportBug.tsx [hunk 1]
```diff
@@ -10,13 +10,34 @@ import {
...
+      {
+        name: "angular",
+        location: "/@abgov/angular-components"
+      },
+      {
+        name: "react",
+        location: "/@abgov/react-components"
+      }
+    ],
+  )
```
Comment (chrisolsen) [inline src/routes/get-started/ReportBug.tsx:40]: This doesn't need to be in a `useState`

File: src/routes/get-started/ReportBug.tsx [hunk 2]
```diff
@@ -10,13 +10,34 @@ import {
   GoATextArea
 } from "@abgov/react-components";
 import { ComponentContent } from "@components/component-content/ComponentContent.tsx";
-import { useState } from "react";
+import { useEffect, useState } from "react";
 import { Octokit } from "octokit";
 import { useNavigate } from "react-router-dom";
   
 export default function ReportBugPage() {
+  interface IPackage {
```
Comment (chrisolsen) [inline src/routes/get-started/ReportBug.tsx:18]: Not a big issue, but no need for the  C# style of defining interfaces. `Package` is fine

File: src/routes/get-started/ReportBug.tsx [hunk 3]
```diff
@@ -180,6 +206,26 @@ export default function ReportBugPage() {
     return valid;
   }
 
+  useEffect(() => {
```
Comment (chrisolsen) [inline src/routes/get-started/ReportBug.tsx:209]: You could add the `async` to line `209`. The only downfall is if you do you can't return the "cleanup" function, but in this case you don't have one, so it's fine. Then you don't have to create a function and manually call it.

**Comment by lizhuomeng71** [comment]
This issue is verified

File: src/routes/get-started/ReportBug.tsx (hunk 1 above)
Comment (ArakTaiRoth) [reply src/routes/get-started/ReportBug.tsx:40]: I had originally written this with a `setPackages` but then it was never used, so I just removed it, not thinking about changing the whole declaration. This is done.

File: src/routes/get-started/ReportBug.tsx (hunk 2 above)
Comment (ArakTaiRoth) [reply src/routes/get-started/ReportBug.tsx:18]: Fair, just how I'm used to defining interfaces, done.

File: src/routes/get-started/ReportBug.tsx (hunk 3 above)
Comment (ArakTaiRoth) [reply src/routes/get-started/ReportBug.tsx:209]: I don't think this is correct, `useEffect` is either supposed to return nothing or a clean-up function as you said, it's not supposed to return a Promise, which is what an async function returns. I could turn the inside into a self-invoking function, but that'll only remove one line of code, and I think makes things slightly less readable, thoughts?

Comment (chrisolsen) [reply src/routes/get-started/ReportBug.tsx:207]: When you call the async function as I mentioned it is returning nothing (it's not returning a promise), so it is ok. As I mentioned if you need to return a cleanup then it doesn't work and you would have to do something like you are doing.

**Pull request merged**